### ISSUE TITLE
/balancetop not working

### ISSUE THREAD

**Issue by Shane1199** [Erigitic/TotalEconomy#126]
My Network is using 1.5.2 on MC 1.10.2, when doing /balancetop it states "Error occurred while executing command: Invalid UUID string: Spawn".

**Comment by MarkL4YG** [comment]
Could you upload your "accounts.conf" to [pastebin](http://pastebin.com) or a similar service?

**Comment by Shane1199** [comment]
http://pastebin.com/FfLUDbAC

**Comment by MarkL4YG** [comment]
```
Spawn {
    dollar-balance=129000
}
```
Remove this entry and it should be working again.
Not sure how this got there though 🤔

**Comment by Erigitic** [comment]
That is very odd...but doing what @MarkL4YG should fix the problem. Let us know if all is well after the fix. Thanks.

**Comment by Shane1199** [comment]
So I removed it, and /sponge plugins reload did nothing. Once I restarted, then it gave the same error except stating Nations rather than spawn. I do suspect the Nations sponge plugin may have something to do with it, as we do have "nation" made for Spawn.

But this popped back:

```
Spawn {
    dollar-balance=129000
}
```

And after removal, it has that same nation error: Error occured while executing command: Invalid UUID string: nation-3e4f264f-cb5c-453e-9f1d-b8d1c4af66a9 And we do have a few accounts apparently made for each nation/town.

Maybe helpful info:
Plugins List: 
Nucleus Mixins (nucleus-mixin) version 0.15.0-5.0
AutoBroadcaster (autobroadcaster) version 0.2.1
NuVotifier (nuvotifier) version 2.3.2.2
PermissionManager (permissionmanager) version 2.5.3
Enjin Minecraft Plugin (enjin-minecraft-plugin) version 3.0.16-sponge
EntityDisguise (edisguise) version 0.5
Pixelmon (pixelmon) version 5.0.0
Gym Wars 2: Feebas Unleashed (feebasunleashed) version 1.2.1
Holograms (holograms) version v2.0
HyperModBanner (hypermodbanner) version 3.2.1
IPRegister (ipregister) version 0.0.3
lootcrate project (lootcrate) version 1.1
MiscStuff (miscstuff) version 1.0-SNAPSHOT
Morpheus (Morpheus) version 1.10.2-3.1.13
Mystery Gift (mysterygift) version 0.3.1
Nations (nations) version 2.2
Nucleus (nucleus) version 0.18.0-5.0
CustomCommands (customcommands) version 1.0
Pixel Announcer (pixelannouncer) version 1.0.1
Pixel Auction (pixelauction) version 1.1.2
Pixel Extras (pixelextras) version 2.2.4
Pixelmon Friends (pixelmonfriends) version 1.0.0
Pixel Shout (pixelshout) version 1.1.2
Prism (prism) version 3.0.0
SafePlace (safeplace) version 2.2.1
Discord Bridge (discordbridge) version 2.2.0
Total Economy (totaleconomy) version 1.5.2
Wondertrade (wondertrade) version 4.4.5

Minecraft: 1.10.2
SpongeAPI: 5.1.0-SNAPSHOT-e221622
SpongeForge: 1.10.2-2171-5.1.0-BETA-2010
Forge: 12.18.3.2185

**Comment by MarkL4YG** [comment]
It may be that the nation plugin does not use UUIDs when doing Economy stuff 🤔

**Comment by Shane1199** [comment]
Yes, this is what the Accounts.conf looks like. You can see it at the bottom. The Spawn thing hasn't appeared since it started, but probably will appear after the next one. http://pastebin.com/VJ5Ri1Zn Anything I should do, they can do, or that you can do to fix /baltop?

**Comment by MarkL4YG** [comment]
@Erigitic could add a fallback for when the account name is not an UUID to commands like baltop.
(Especially because ``nation-<UUID>`` should cause this exception too and non-player accounts would always have to have a resolvable UID)

**Comment by Erigitic** [comment]
I think this could be fixed by adding in support for virtual accounts.

**Comment by Erigitic** [comment]
I'm going to close this issue as it will be fixed by #130
